projects/1.16.1/assets/woot/woot/lang/zh_cn.json

```
@@ -0,0 +1,535 @@
+  "item.woot.ench_plate_2": "达人之板",
+  "item.woot.ench_plate_3": "魔法师之板",
+  "item.woot.enchant_fluid_bucket": "液态魔咒桶",
+  "item.woot.gray_dyecasing": "灰色染料框架",
+  "item.woot.gray_dyeplate": "灰色染料板",
+  "item.woot.green_dyecasing": "绿色染料框架",
+  "item.woot.green_dyeplate": "绿色染料板",
+  "item.woot.guide.landing": "一本写给这大的，坏的，魔法多方块生物工厂的指南。就像烹饪锅一样，你会喜欢它或者讨厌它的--由你选择。",
+  "item.woot.guide.name": "Woot指南",
+  "item.woot.hammer": "YA锤子",
+  "item.woot.headless_1": "从“头”开始 I 升级插件",
+  "item.woot.headless_2": "从“头”开始 II 升级插件",
+  "item.woot.headless_3": "从“头”开始 III 升级插件",
+  "item.woot.intern": "工厂搭建器",
+  "item.woot.light_blue_dyecasing": "淡蓝色染料框架",
+  "item.woot.light_blue_dyeplate": "淡蓝色染料板",
+  "item.woot.light_gray_dyecasing": "淡灰色染料框架",
+  "item.woot.light_gray_dyeplate": "淡灰色染料板",
+  "item.woot.lime_dyecasing": "黄绿色染料框架",
+  "item.woot.lime_dyeplate": "黄绿色染料板",
+  "item.woot.looting_1": "金银岛 I 升级插件",
+  "item.woot.looting_2": "金银岛 II 升级插件",
+  "item.woot.looting_3": "金银岛 III 升级插件",
+  "item.woot.machine_casing": "机器框架",
+  "item.woot.magenta_dyecasing": "品红色染料框架",
+  "item.woot.magenta_dyeplate": "品红色染料板",
+  "item.woot.mass_1": "横冲直撞 I 升级插件",
+  "item.woot.mass_2": "横冲直撞 II 升级插件",
+  "item.woot.mass_3": "横冲直撞 III 升级插件",
+  "item.woot.mob_essence_fluid_bucket": "净化精华桶",
+  "item.woot.mobshard": "生物碎片",
+  "item.woot.orange_dyecasing": "橙色染料框架",
+  "item.woot.orange_dyeplate": "橙色染料板",
+  "item.woot.pink_dyecasing": "粉红色染料框架",
+  "item.woot.pink_dyeplate": "粉红色染料板",
+  "item.woot.plate_die": "板模具",
+  "item.woot.prism": "棱镜",
+  "item.woot.puredye_fluid_bucket": "纯净染料桶",
+  "item.woot.purple_dyecasing": "紫色染料框架",
+  "item.woot.purple_dyeplate": "紫色染料板",
+  "item.woot.rate_1": "疯狂捕食 I 升级插件",
+  "item.woot.rate_2": "疯狂捕食 II 升级插件",
+  "item.woot.rate_3": "疯狂捕食 III 升级插件",
+  "item.woot.red_dyecasing": "红色染料框架",
+  "item.woot.red_dyeplate": "红色染料板",
+  "item.woot.shard_die": "碎片模具",
+  "item.woot.si_dust": "地狱铁粉",
+  "item.woot.si_ingot": "地狱铁锭",
+  "item.woot.si_plate": "地狱铁板",
+  "item.woot.slaughter_1": "翻滚血肉玩偶 I 升级插件",
+  "item.woot.slaughter_2": "翻滚血肉玩偶 II 升级插件",
+  "item.woot.slaughter_3": "翻滚血肉玩偶 III 升级插件",
+  "item.woot.t1shard": "青瓷精华",
+  "item.woot.t2shard": "天蓝精华",
+  "item.woot.t3shard": "拜占庭精华",
+  "item.woot.tier_shard_1": "攀登高峰 I 升级插件",
+  "item.woot.tier_shard_2": "攀登高峰 II 升级插件",
+  "item.woot.tier_shard_3": "攀登高峰 III 升级插件",
+  "item.woot.white_dyecasing": "白色染料框架",
+  "item.woot.white_dyeplate": "白色染料板",
+  "item.woot.xp_1": "智慧 I 升级插件",
+  "item.woot.xp_2": "智慧 II 升级插件",
+  "item.woot.xp_3": "智慧 III 升级插件",
+  "item.woot.xpshard": "经验碎片",
+  "item.woot.xpsplinter": "经验尖片",
+  "item.woot.yellow_dyecasing": "黄色染料框架",
+  "item.woot.yellow_dyeplate": "黄色染料板",
+  "itemGroup.woot": "Woot",
+  "jei.woot.anvil.0": "用于制造控制器。",
+  "jei.woot.anvil.1": "必须放置于岩浆块上。",
+  "jei.woot.anvil.2": "右键单击砧座顶部以放置物品。",
+  "jei.woot.anvil.3": "使用YA锤子以制造。",
+  "jei.woot.intern.0": "可用于自动建造或检验工厂。",
+  "jei.woot.intern.1": "对着工厂核心使用以建造或检验",
```

|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
  "jei.woot.intern.1": "对着工厂核心使用，可以建造或检验工厂结构",
```
拆句，to do 意味着去做某些事情，这个时候只用“以”引导后面的句子，不是不可以，但是汉语中不常用这种表达，口语中习惯于拆句子，将要做的事情单独列出。
例如：
> 我把苹果削了皮，这样方便我吃苹果。

但不会说：

> 我把苹果先削了皮以方便我吃苹果。